frontend/sozisel-app/src/components/Files/FileCard/FileCard.tsx

```diff
@@ -35,47 +35,45 @@ export default function FileCard({
   const { t } = useTranslation("common");
 
   return (
-    <>
-      <div className="FileCard">
-        <div className="fileCardContent">
-          <div className="cardHeader">
-            <IconButton
-              href={`${getServerUrl()}:4000/session_resource/${path}`}
-              target="_blank"
-            >
-              <GetAppIcon />
+    <div className="FileCard">
+      <div className="fileCardContent">
+        <div className="cardHeader">
+          <IconButton
+            href={`${getServerUrl()}:4000/session_resource/${path}`}
```

Comment:
Can you create a helper module that will be able to resolve all existing external links so we have a single place where we can change stuff and don't have to remember all ports and other things? When you get that ready please try to find all urls and replace them.

Reply:
Will be helpful for deployment